ocs_ci/ocs/ui/views.py

```diff
@@ -171,6 +171,13 @@
 }
 
 locators = {
+    "4.8": {
+        "login": login,
+        "page": page_nav,
+        "deployment": deployment_4_7,
```

Comment:
Does this line mean that all the locators for 4.8 deployment are the same as 4.7 locators? Do we then need this special 4.8 dictionary?

Reply:
@ebenahar I tested lso cluster +Internal cluster on OCS4.8 and OCP 4.8,and the locators for 4.8 deployment are the same as 4.7